README.md

```diff
@@ -83,6 +83,7 @@ No provider.
 | core\_timeout\_duration\_minutes | Spot provisioning timeout for core instances, in minutes | `number` | `10` | no |
 | create\_static\_cluster | True if the module should create a static cluster. False if the module should create supporting infrastructure but not the cluster itself. | `bool` | `true` | no |
 | custom\_ami\_id | The ID of a custom Amazon EBS-backed Linux AMI | `string` | `null` | no |
+| security\_configuration | The name of an EMR Security Configuration | `string` | `null` | no |
```

Comment:
@olivito have you been making these files by hand?

If you run `make terraform/docs` it'll generate these tables for you.

Found in this PR that this was out of order:
https://github.com/Datatamer/terraform-aws-emr/pull/64/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5